app/main.py

```diff
@@ -1,3 +1,23 @@
-def shop_trip():
-    # write your code here
-    pass
+from json import load
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("config.json") as config:
+        data = load(config)
+
+    fuel_price = data["FUEL_PRICE"]
+
+    for customer in data["customers"]:
+        current_customer = Customer(*customer.values())
+        shops = [Shop(*shop.values()) for shop in data["shops"]]
```

Comment:
Similarly, ensure that the `Shop` class constructor is correctly handling the unpacked values from the `shop` dictionary. The order of values should match the parameters expected by the `Shop` class constructor.